### ISSUE TITLE
Inconsistent nonce and IV usage

### ISSUE THREAD

**Issue by chris-wood** [sframe-wg/sframe#155]
The spec seems to use nonce and IV inconsistently, especially when talking about nonce reuse. It's probably better if the term was made consistent. I would choose nonce.

**Comment by bifurcation** [comment]
Agreed we need to pick one.  The only possible exception might be when we are talking about AES-CTR, which is specified in terms of an IV.

I would also choose nonce.
